sentry_sdk/tracing.py

```diff
@@ -1582,6 +1582,9 @@ def is_success(self):
 
     def finish(self, end_timestamp=None):
         # type: (Optional[Union[float, datetime]]) -> Optional[str]
+        if hasattr(self._otel_span, "status") and self._otel_span.status.status_code == StatusCode.UNSET:
```

Comment:
let's do this in `__exit__` above since we are already setting `INTERNAL_ERROR` there.

Reply:
But if people do not use the contextmanager, the status will not be set.

Reply:
already doing this in another branch so you can just remove this and pull later

Reply:
ok. removing it.